### PR TITLE
docs/esp32/quickref: Note about automatic WLAN reconnect.

### PR DESCRIPTION
**Pull request by karfas**
According to issue #5326, the ESP32 will attempt to connect() to an access point **forever**.
This was merged in b80bccc and had been undocumented since then (mid 2019).

### CODE REVIEW

File: docs/esp32/quickref.rst [hunk 1]
```diff
@@ -102,6 +106,7 @@ Once the network is established the :mod:`socket <usocket>` module can be used
 to create and use TCP/UDP sockets as usual, and the ``urequests`` module for
 convenient HTTP requests.
 
+
```
Comment (kadamski) [inline docs/esp32/quickref.rst:109]: This additional whiteline seems unrelated to the rest of the commit. Was it done by mistake?

Comment (karfas) [reply docs/esp32/quickref.rst:109]: Is a mistake, sure. However, this PR is most likely obsoleted by #7348.

**Comment by dpgeorge** [comment]
This was merged as part of 364670ecf10b2f31e9a207bc440cfb731dcfa603
